### ISSUE TITLE
Surface required delimiters for Targets' and Actions' syntax patterns

### ISSUE THREAD

**Issue by francisfuzz** [github/catalyst#319]
## Summary

The [`Targets > Target Syntax`](https://catalyst.rocks/guide/targets/#target-syntax) documentation covers `controller` and `target` but doesn't explicitly call out its required delimiter, `.`. Similarly, the [`Actions > Actions Syntax`](https://catalyst.rocks/guide/actions/#actions-syntax) documentation defines its pattern, but doesn't call out its required delimiters (`:` and `#` respectively).

As a first-time reader interested in using Catalyst, I think it would be helpful to notate required delimiter, so folks don't think it's option and use other characters expecting it to work. 😅 

For example, I would imagine having this called out in targets like so:

> The target syntax follows a pattern of `controller.target`.

> - `controller` must be the name of a `controller` ascendant to the element.
> - `.` must be a period as the delimiter between `controller` and `target` `<= the new thing!`
> - `target` must be the name matching that of a `@target` (or `@targets`) annotated field within the Controller code.

**Comment by francisfuzz** [comment]
@keithamus - Hi! I'm sorry I couldn't find a more specific team to reach out to, but if this is a welcome contribution, I'd be happy to take it on 🙇

**Comment by keithamus** [comment]
Hey @francisfuzz thanks for raising this! 

Firstly I think it's worth pointing out that Catalyst is not actively maintained and I doubt it will be in any capacity in the foreseeable future. If you're interested in a web component framework that is more actively maintained I'd point you towards [Lit](https://lit.dev). If you're still interested in Catalyst, then you are welcome to fork the code (it's MIT licensed) and maintain your own fork of it, which is likely how any new development would take place.

With that out of the way, I agree with the point you raised and would enjoy seeing some improvements to the documentation and I'd be happy to merge any PRs which evolve the documentation of the existing code & featureset.

**Comment by francisfuzz** [comment]
@keithamus - Thank you for sharing that context! ✨ 

😱 Today I learned that Catalyst is no longer actively maintained. I did not know prior to opening this! 😞

This issue came by way of me learning Catalyst to make updates to an existing feature for work, rather than for my own personal project. I'll certainly reach out for [Lit](https://lit.dev/) if it comes down to doing something completely new in the future! 💡 

> With that out of the way, I agree with the point you raised and would enjoy seeing some improvements to the documentation and I'd be happy to merge any PRs which evolve the documentation of the existing code & featureset.

Roger that! I'll self-assign, carve out some time to work on this, and follow up once I get more cycles. Thanks, Keith!
